pkg/auth/rhsso_authz_handler.go

```diff
@@ -179,17 +177,17 @@ func (a *AuthzHandler) checkInfraEnvBasedAccess(id string, request *http.Request
 	return a.isObjectOwnedByUser(id, &common.InfraEnv{}, payload)
 }
 
-func (a *AuthzHandler) Authorizer(request *http.Request) error {
+func (a *AuthzHandler) AuthorizerFunc(request *http.Request) error {
```

Comment:
Why change these names?

Reply:
Authorizer is now the interface for the entire service so this name is overloaded. After the change I found the old name confusing, so I wanted to convey that this is just the authorizer callback function for the resapi middleware. If we you have a better name please suggest it.

Reply:
I would only use `Func` in a function name if we're returning a function type, which this isn't.

I think `Authorize` would be better since this is actually doing the work of authorizing the request.
Also please remove `Func` from the other functions as well.